### PR TITLE
[homekit] support stop for blinds

### PR DESCRIPTION
**Pull request by yfre**
home app never sends "STOP" but always sends the target position, e.g. 100% or 0% for fully open or closed blinds. 
Rollershutter in openHAB and some device do support stop. 
this PR adds emulation of stop if user click on blind icons in the home app while blinds is moving. 
report and discussion here
https://community.openhab.org/t/oh3-rollershutter-and-apple-homekit/112376/31

this PR also fix one potential NPE and a typo in the docu "light -> lighting"

### CODE REVIEW

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.io.homekit/README.md [hunk 1]
```diff
@@ -310,7 +310,19 @@ In case you need to disable this logic you can do it with configuration paramete
 Rollershutter window_covering "Window Rollershutter" {homekit = "WindowCovering"  [inverted=false]}
 Rollershutter window          "Window"               {homekit = "Window" [inverted=false]}
 Rollershutter door            "Door"                 {homekit = "Door" [inverted=false]}
+ ```
+
+HomeKit home app never sends STOP" but only the target position. 
+If you add configuration parameter "stop=true", openHAB will emulate stop and send "STOP" command to rollenshutter item if you click on the blind icon in the home app while the blind is moving.
```
[the remark this answers is not in the document]
Comment (yfre) [reply bundles/org.openhab.io.homekit/README.md]: the apple app on iphone / ipad for HomeKit 
is iOS home app better. or maybe HomeKit home app..

**Review by lolodomo — APPROVED** [review]
LGTM

**Comment by lolodomo** [comment]
@ccutrer : would you like to review?

**Review by ccutrer — APPROVED** [review]
LGTM

**Comment by ccutrer** [comment]
> @ccutrer : would you like to review?

Even though I'm in CODEOWNERS now, I was not notified of the PR until you tagged me, and I'm still not showing as a code owner in the GitHub UI. Do you know if I just need to be added to the openHAB GitHub organization? Or do I need to actually be added to this specific repo with some amount of access? And if so, can you do that, or does it need to be someone else?

**Comment by lolodomo** [comment]
> Even though I'm in CODEOWNERS now, I was not notified of the PR until you tagged me, and I'm still not showing as a code owner in the GitHub UI. Do you know if I just need to be added to the openHAB GitHub organization? Or do I need to actually be added to this specific repo with some amount of access? And if so, can you do that, or does it need to be someone else?

Normally there is nothing to do, but there is apparently a problem with your account, for example if I try to filter PR by author, I do not find you in the list. Maybe @jlaur or @fwolter has an idea what's wrong?

**Comment by jlaur** [comment]
@ccutrer is not a member of the contributors group, @kaikreuzer can you add?

**Comment by lolodomo** [comment]
> @ccutrer is not a member of the contributors group, @kaikreuzer can you add?

There is a manual action to do each time a new contributor contributes?

**Comment by kaikreuzer** [comment]
Done.

> There is a manual action to do each time a new contributor contributes?

@lolodomo Yes, indeed - everybody needs to be added to this group: https://github.com/orgs/openhab/teams/contributors

**Comment by ccutrer** [comment]
Thanks @kaikreuzer!

**Pull request merged**